### PR TITLE
Remove unused fields of Node

### PR DESCRIPTION
**Pull request by Mykhailenko**
so client side will store less -> improve performance

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 37 issues
* ![CRITICAL][CRITICAL] 10 critical
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 22 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [NodeSource.java#L393](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java#L393): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![CRITICAL][CRITICAL] [RMController.java#L282](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L282): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [RMController.java#L285](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L285): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [RMController.java#L350](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L350): Refactor this method to reduce its Cognitive Complexity from 25 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
1. ![CRITICAL][CRITICAL] [RMController.java#L684](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L684): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![CRITICAL][CRITICAL] [RMController.java#L706](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L706): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![CRITICAL][CRITICAL] [RMController.java#L910](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L910): Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
1. ![CRITICAL][CRITICAL] [RMController.java#L1004](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L1004): Define a constant instead of duplicating this literal "</strong>" 4 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [RMController.java#L1074](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L1074): Define a constant instead of duplicating this literal "Confirm removal of <strong>" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [RMController.java#L1292](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L1292): Define a constant instead of duplicating this literal "cause" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Mykhailenko** [comment]
retest this please

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java [hunk 1]
```diff
@@ -235,21 +229,14 @@ public String getId() {
...
-
-            private String nodeLocker = null;
-
             private String eventType = null;
 
-            Node(String nodeUrl, String nodeState, String nodeInfo, long timeStamp, String timeStampFormatted,
-                    String nodeProvider, String nodeOwner, String sourceName, String hostName, String vmName,
-                    String description, String defaultJMXUrl, String proactiveJMXUrl, boolean isLocked, long lockTime,
-                    String nodeLocker, String eventType) {
+            Node(String nodeUrl, String nodeState, long timeStamp, String nodeProvider, String nodeOwner,
```
Comment (sonarqube-activeeon) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java]: MAJOR Constructor has 13 parameters, which is greater than 7 authorized. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 38 issues
* ![CRITICAL][CRITICAL] 10 critical
* ![MAJOR][MAJOR] 5 major
* ![MINOR][MINOR] 23 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [NodeSource.java#L350](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java#L350): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![CRITICAL][CRITICAL] [RMController.java#L282](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L282): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [RMController.java#L285](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L285): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [RMController.java#L350](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L350): Refactor this method to reduce its Cognitive Complexity from 25 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
1. ![CRITICAL][CRITICAL] [RMController.java#L684](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L684): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![CRITICAL][CRITICAL] [RMController.java#L706](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L706): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![CRITICAL][CRITICAL] [RMController.java#L892](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L892): Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
1. ![CRITICAL][CRITICAL] [RMController.java#L986](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L986): Define a constant instead of duplicating this literal "</strong>" 4 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [RMController.java#L1056](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L1056): Define a constant instead of duplicating this literal "Confirm removal of <strong>" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [RMController.java#L1274](https://github.com/ow2-proactive/scheduling-portal/blob/1706d4b247ec79e0b3cddf273f1eada18838156d/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java#L1274): Define a constant instead of duplicating this literal "cause" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Mykhailenko** [comment]
Fabien explained me that it was not so good idea,

**Pull request closed without merging**